pkg/config/default.go

```diff
@@ -398,6 +399,9 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 
 	c.PodExitPolicy = defaultPodExitPolicy
 
+	dirname := homedir.Get()
+	c.SSHConfig = filepath.Join(dirname, ".ssh", "config")
```

Comment:
This should be a function in the ssh bindings.

c.SSHConfig=ssh.Config()

Reply:
Should respect a CONTAINERS_SSH_CONFIG environment variable which could be used for testing.

Reply:
hm, it seems like this causes a bad import cycle. I will make it a separate function in this file and check for the env variable